target/linux/mvebu/files/arch/arm64/boot/dts/marvell/armada-3720-gl-mv1000.dts

```diff
@@ -162,18 +162,24 @@
 				reg = <1>;
 				label = "wan";
 				phy-handle = <&switch0phy0>;
+
+				mtd-mac-address = <&factory 0x0>;
```

Comment:
Won't this be inherited from eth0?

Reply:
Yes, but I wanted to have it defined the same way for all ports

Reply:
We typically don't do that, as it's redundant (DRY). However, my experience is mostly based on other targets, e.g. ramips/mt7621 where we set &gmac and then just the wan-port if it's different.

Reply:
ok, i've dropped it